backendRoot/src/main/java/com/virtuslab/gitmachete/backend/root/GitMacheteRepositoryBuilder.java

```diff
@@ -67,6 +60,11 @@ public GitMacheteRepositoryBuilder(Path pathToRepoRoot) {
     this.pathToRepoRoot = pathToRepoRoot;
   }
 
+  public GitMacheteRepositoryBuilder branchLayout(IBranchLayout givenBranchLayout) {
```

Comment:
Lombok-generated code marks the param (now called `givenBranchLayout`) as `@MonotonicNonNull` as well... and one can't assign a `@MonotonicNonNull` value to another `@MonotonicNonNull` value

Reply:
Coz `@MonotonicNonNull` is essentially:

* `@Nullable` when getting
* `@NonNull` when setting